components/datetime/src/scaffold/fieldset_traits.rs

```diff
@@ -45,6 +50,11 @@ pub trait DateInputMarkers: UnstableSealed {
 /// (data markers only).
 ///
 /// This is a sealed trait implemented on field set markers.
+///
+/// <div class="stab unstable">
+/// 🚧 This trait is considered unstable; it may change at any time, in breaking or non-breaking ways,
+/// including in SemVer minor releases. Do not implement this trait in userland.
```

Comment:
It should be more like "do not implement this trait in userland _unless_ you know what you are doing and are ready to fix your code even in minor releases" but I don't know how to say that concisely.